scripts/chart/promote-to-stable.sh

```diff
@@ -1,32 +1,37 @@
 #!/bin/bash
 
+drone_cli_help()
+{
+    echo "  Requires Drone CLI and personal access token"
+    echo "  Download CLI following instructions at https://docs.drone.io/cli/install/"
+    echo "  Then configure access token: https://docs.drone.io/cli/setup/"
+    echo ""
+    echo "  To test if Drone CLI is properly configured type:"
+    echo ""
+    echo "     drone build ls rancher/rancher"
+    echo ""
+    echo "  This will show the last 25 builds"
+}
+
 if [[ -z "${1}" ]]; then
     echo "Promote a chart in latest to stable."
     echo "  $0 <tag>"
     echo ""
-    echo "  Requires a GitHub Personal Access Token exported as GH_TOKEN"
-    echo "    Create a token at https://github.com/settings/tokens"
-    echo "    Permissions: repo_deployment"
+    drone_cli_help
     exit 1
 fi
 
-if [[ -z "${GH_TOKEN}" ]]; then
-    echo 'ERROR: Missing $GH_TOKEN'
-    echo 'Create a GitHub Personal Access Token and export it as GH_TOKEN'
-    echo 'Required permissions: repo_deployment'
-    echo 'https://github.com/settings/tokens'
+if ! drone -v; then
+    drone_cli_help
     exit 1
 fi
 
-data=$(cat <<EOF
-{
-    "ref": "refs/tags/${1}",
-    "environment": "promote-stable",
-    "auto_merge": false,
-    "description": "promote-to-stable.sh"
-}
-EOF
-)
+build_number=$(drone build ls cloudnautique/ci-cdtesting --event tag --format "{{.Number}},{{.Ref}}"| grep ${1} |cut -d',' -f1)
```

Comment:
Should be `drone build ls rancher/rancher`?  Does this have a limit on how many builds it shows.  What happens if we want to promote something that is a couple weeks old? In theory there's supposed to be a soak period between publish to latest and stable.